zend/classimpl.cpp

```diff
@@ -1313,6 +1327,15 @@ const struct _zend_function_entry *ClassImpl::entries()
     // already initialized?
     if (_entries) return _entries;
 
+    // if the class is serializable
+    if (_base->serializable())
+    {
+        // add the serialize method if the class does not have one defined yet
+        if (!hasMethod("serialize")) method("serialize");
+        // add the unserialize method if the class does not have one defined yet
+        if (!hasMethod("unserialize")) method("unserialize", 0, {Php::ByVal("serialized")});
+    }
```

Comment:
These calls seem to add _abstract_ methods to the class, we need _real_ methods that also work